conda_forge_tick/utils.py

```diff
@@ -7,6 +7,15 @@
 from conda_build.metadata import parse
 
 
+class UniversalSet(set):
```

Comment:
Why is this needed? (Not saying it's wrong, I just can't see it yet)

Reply:
This is the default value for requirements we remove pinnings on. If no specific packages are provided, just remove the pinnings on everything. Since the filter and migrator use set intersection, this is a nice way to make it a no op.

Reply:
Also, you should not subclass from set, but rather from collections.abc.Set